pkg/config/cmd/cmd.go

```diff
@@ -46,13 +55,12 @@ func (b *Bulk) Create(list *kapi.List, namespace string) []error {
 			after(info, err)
 			continue
 		}
-		data, err := info.Mapping.Codec.Encode(item)
-		if err != nil {
-			errs = append(errs, err)
-			after(info, err)
```

Comment:
the error is still appended to errs and after is still called here: 
https://github.com/openshift/origin/pull/4613/files#diff-7ac827ae32eef0a0caf4198518f4fcd1L57